demo/list-box-presentation-demos.html

```diff
@@ -0,0 +1,86 @@
+<dom-module id="list-box-presentation-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+    <h3>Separators</h3>
+    <vaadin-demo-snippet>
+      <template preserve-content>
+        <vaadin-list-box>
+          <vaadin-item disabled>Weekdays</vaadin-item>
+          <hr>
+          <vaadin-item>Monday</vaadin-item>
+          <vaadin-item>Tuesday</vaadin-item>
+          <vaadin-item>Wednesday</vaadin-item>
+          <vaadin-item>Thursday</vaadin-item>
+          <vaadin-item>Friday</vaadin-item>
+          <hr>
+          <vaadin-item>Saturday</vaadin-item>
+          <vaadin-item>Sunday</vaadin-item>
+        </vaadin-list-box>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Customizing listbox item label</h3>
+    <vaadin-demo-snippet>
+      <template preserve-content>
+        <b>User</b>
+        <vaadin-list-box>
+          <vaadin-item value="1" selected>
+            <img style="width: 21px; border-radius: 50%; vertical-align: bottom;"
```

Comment:
These inline `style` attributes are needed for `vaadin-select` but not for `vaadin-list-box`.

You can remove them and use the following instead (inside of `<template>`):

```html
<style>
  img {
    width: 21px; 
    border-radius: 50%; 
    vertical-align: bottom;
  }
</style>
```

Reply:
Done.